src/schema/schema.h

```diff
@@ -22,38 +21,33 @@ typedef enum {
  * similar to a relational table structure, our schemas are a collection
  * of attributes we've encountered overtime as entities were created or updated. */
 typedef struct {
-  int id;                 /* Internal ID to a matrix within the graph. */
-  char *name;             /* Schema name. */
-  Index** indices;        /* Indices applicable to schema. */
-  RSIndex* fulltextIdx;   /* Full-text index. */
+  int id;               // Internal ID to a matrix within the graph.
+  char *name;           // Schema name.
+  Index* index;         // Exact match index.
+  Index* fulltextIdx;   // Full-text index.
```

Comment:
This change is the cause of the current Circle error. It's important to note that serialization requires significant (but I think not breaking) updates to accommodate these changes.

Reply:
We'll go with two decoders, for backward compatibility.